lib/heimdall_tools/nessus_mapper.rb

```diff
@@ -96,14 +117,15 @@ def nist_tag(pluginfamily, pluginid)
     end
 
     def impact(severity)
+      # Map CAT levels and Plugin severity to HDF impact levels
       case severity
       when "0"
         IMPACT_MAPPING[:Info]
-      when "1"
+      when "1","III"
```

Comment:
This is incorrect, a CAT I or I is high ...
A CAT III or 3 is low

Reply:
Yep thats the CAT logic coded here
"III" -> low
"II" -> medium
"I" -> high

Reply:
The method is designed to make it work with both plugin severity and Nessus CAT reference.

Reply:
Ok